File: SDKs/googleapis.com/osconfig/v1alpha/java/lib/src/main/java/openapisdk/models/shared/ListInstanceOsPoliciesCompliancesResponse.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * ListInstanceOsPoliciesCompliancesResponse
 * A response message for listing OS policies compliance data for all Compute Engine VMs in the given location.
**/
public class ListInstanceOsPoliciesCompliancesResponse {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("instanceOsPoliciesCompliances")
    public InstanceOsPoliciesCompliance[] instanceOsPoliciesCompliances;
    public ListInstanceOsPoliciesCompliancesResponse withInstanceOsPoliciesCompliances(InstanceOsPoliciesCompliance[] instanceOsPoliciesCompliances) {
        this.instanceOsPoliciesCompliances = instanceOsPoliciesCompliances;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("nextPageToken")
    public String nextPageToken;
    public ListInstanceOsPoliciesCompliancesResponse withNextPageToken(String nextPageToken) {
        this.nextPageToken = nextPageToken;
        return this;
    }
}

File: SDKs/googleapis.com/osconfig/v1alpha/python/src/sdk/models/shared/listinstanceospoliciescompliancesresponse.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import instanceospoliciescompliance as shared_instanceospoliciescompliance


@dataclass_json
@dataclasses.dataclass
class ListInstanceOsPoliciesCompliancesResponse:
    r"""ListInstanceOsPoliciesCompliancesResponse
    A response message for listing OS policies compliance data for all Compute Engine VMs in the given location.
    """
    
    instance_os_policies_compliances: Optional[list[shared_instanceospoliciescompliance.InstanceOsPoliciesCompliance]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('instanceOsPoliciesCompliances') }})
    next_page_token: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('nextPageToken') }})
    
